CODE_OF_CONDUCT.md

```diff
@@ -1,49 +1,133 @@
-permanently any contributor for other behaviors that they deem inappropriate,
-threatening, offensive, or harmful.
+## Scope
 
-By adopting this Code of Conduct, project maintainers commit themselves to
-fairly and consistently applying these principles to every aspect of managing
-this project. Project maintainers who do not follow or enforce the Code of
-Conduct may be permanently removed from the project team.
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
 
-This code of conduct applies both within project spaces and in public spaces
-when an individual is representing the project or its community.
+## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting a project maintainer at hola@decidim.org. All
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. Maintainers are
-obligated to maintain confidentiality with regard to the reporter of an
-incident.
+reported to the [Coordination Committee][committee] responsible for enforcement at
+associacio@decidim.org.
+All complaints will be reviewed and investigated promptly and fairly.
+
+All the [Coordination Committee][committee] members are obligated to respect the privacy and security of the
+reporter of any incident.
+
+## Enforcement Guidelines
+
+The [Coordination Committee][committee] will follow these Community Impact Guidelines in determining
+the consequences for any action they deem in violation of this Code of Conduct:
+
+### 1. Correction
+
+**Community Impact**: Use of inappropriate language or other behavior deemed
+unprofessional or unwelcome in the community.
+
+**Consequence**: A private, written warning from the [Coordination Committee][committee], providing
+clarity around the nature of the violation and an explanation of why the
+behavior was inappropriate. A public apology may be requested.
+
+### 2. Warning
+
+**Community Impact**: A violation through a single incident or series of
+actions.
+
+**Consequence**: A warning with consequences for continued behavior. No
+interaction with the people involved, including unsolicited interaction with
+those enforcing the Code of Conduct, for a specified period of time. This
+includes avoiding interactions in community spaces as well as external channels
+like social media. Violating these terms may lead to a temporary or permanent
+ban.
+
+### 3. Temporary Ban
+
+**Community Impact**: A serious violation of community standards, including
+sustained inappropriate behavior.
+
+**Consequence**: A temporary ban from any sort of interaction or public
```

Comment:
Where would someone find a list of people who are currently or have been banned, whether temporary or permanent?

Reply:
As [mentioned](https://github.com/decidim/decidim/pull/12525#issuecomment-2039338821) by @NilHomedes 

> The suggestions made by @RCheesley will be discussed in the upcoming participatory process.

So I'll mark this conversation as resolved so we can proceed with the merge